gb-dbg/src/debugger/disassembler.rs

```diff
@@ -19,23 +19,26 @@ impl DisassemblyViewer {
         let current_opcode = &self.cache[0];
 
         // The "+ 1" delays the cache update so we can keep displaying the current instruction
-        let next_instr_start_address = opcode_len(current_opcode) + pc + 1;
+        let next_instr_start_address =
+            opcode_len(current_opcode) + pc + MCYCLES_CONSUMED_TO_FETCH_INSTRUCTION;
```

Comment:
this expression don't make sense, we're calculating an address not a duration

Reply:
Yes but this `+ 1` is the consequence of having to consume 1 cycle to both fetch the next instruction infos and update the changes in the registers made by the previous opcode.
I could do with some help with the naming if you have a suggestion.

Reply:
now that we can check for the event `Processing new opcode` we could rely on that ?